config/config.go

```diff
@@ -937,6 +938,9 @@ type Config struct {
 	// Section for configuring OpenTracing support
 	Tracer Tracer `json:"tracing"`
 
+	// Section for configuring Opentelemetry
+	OpenTelemetry otelconfig.OpenTelemetry `json:"opentelemetry"`
```

Comment:
am wondering how will we publish the documntation of this config option with the docs generator 🤔

Reply:
It should work the same? it's already included in the schema generation